modules/nw-allocate-load-balancers-to-subnets-procedure.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// *networking/allocating-load-balancers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-allocating-load-balancers-to-subnets-procedure_{context}"]
+= Specifying AWS subnets for OpenShift ingress load balancers at installation
```

Comment:
Should we mention `API` here as well ? like `Specifying AWS subnets for OpenShift API and ingress load balancers at installation` ?